requirements.txt

```diff
@@ -4,7 +4,7 @@ cryptography
 dipy
 futures
 h5py
-ivadomed
+ivadomed>=1.2.1
```

Comment:
I wonder if we want to carefully monitor ivadomed version (ie use `==`) as it is quite critical & lot of changes at the mment